cypress/e2e/orders/inventory-interaction/create-new-holdings.cy.js

```diff
@@ -12,163 +10,160 @@ import NewOrganization from '../../../support/fragments/organizations/newOrganiz
 import OrderLines from '../../../support/fragments/orders/orderLines';
 import ItemRecordView from '../../../support/fragments/inventory/item/itemRecordView';
 import ServicePoints from '../../../support/fragments/settings/tenant/servicePoints/servicePoints';
-import NewLocation from '../../../support/fragments/settings/tenant/locations/newLocation';
+import Locations from '../../../support/fragments/settings/tenant/location-setup/locations';
 import InventoryInstance from '../../../support/fragments/inventory/inventoryInstance';
 import ItemActions from '../../../support/fragments/inventory/inventoryItem/itemActions';
 import { ITEM_STATUS_NAMES } from '../../../support/constants';
 import Users from '../../../support/fragments/users/users';
 import InventoryInstances from '../../../support/fragments/inventory/inventoryInstances';
-import InventoryNewHoldings from '../../../support/fragments/inventory/inventoryNewHoldings';
+import InventoryHoldings from '../../../support/fragments/inventory/holdings/inventoryHoldings';
 import ItemRecordEdit from '../../../support/fragments/inventory/item/itemRecordEdit';
 
 describe('Orders: Inventory interaction', () => {
-  const order = {
-    ...NewOrder.defaultOneTimeOrder,
-    approved: true,
-  };
-  const organization = {
-    ...NewOrganization.defaultUiOrganizations,
-    accounts: [
-      {
-        accountNo: getRandomPostfix(),
-        accountStatus: 'Active',
-        acqUnitIds: [],
-        appSystemNo: '',
-        description: 'Main library account',
-        libraryCode: 'COB',
-        libraryEdiCode: getRandomPostfix(),
-        name: 'TestAccout1',
-        notes: '',
-        paymentMethod: 'Cash',
-      },
-    ],
-  };
   const barcodeForFirstItem = Helper.getRandomBarcode();
   const barcodeForSecondItem = Helper.getRandomBarcode();
 
-  let orderNumber;
-  let user;
-  let effectiveLocationServicePoint;
-  let firstLocation;
-  let instanceRecord = null;
-  let secondLocation;
+  const testData = {
```

Comment:
Sorry, but why you fix my test?and remove some methods???

Reply:
Fixed